index.html

```diff
@@ -190,6 +190,8 @@ <h2>Contact me</h2>
             <input
               type="email"
               id="email"
+              title="The email must be in all lowercase letters."
+              pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
```

Comment:
- [ ] Could you please create a validation using JavaScript? The current pattern you've implemented **doesn't prevent the form from being submitted**, which is not the intended behavior. Currently, **it only displays a message when I hover over the input field.** It would be great if you could make a text appear when I click on the submit button and prevent the form from being submitted if there is an uppercase letter. 👍

   ### Project requirement:
   ![image](https://github.com/Mehr-o-maah/Portfolio_2.github.io/assets/107952724/6b804eae-f813-481c-b626-333cd30cf789)